test/python/test_polyhedron.py

```diff
@@ -86,3 +86,30 @@ def test_polyhedron(
     non_modified_faces = np.array(polyhedron_non_modified.faces)
     np.testing.assert_array_almost_equal(non_modified_faces, polyhedral_source[1])
 
+def test_polyhedron_metric() -> None:
+    """Tests the metric utility displaying the units.
```

Comment:
I take it this shouldn't check values? Just to be sure, then ok

Reply:
Yes, no values just the string values of the members